packages/core/src/app/ovm/deciders/and-decider.ts

```diff
@@ -16,6 +16,15 @@ export interface AndDeciderInput {
  * Decider that decides true iff both of the provided properties evaluate to true.
  */
 export class AndDecider implements Decider {
+  private static _instance: AndDecider
```

Comment:
Ohh, yeah, it's not. Was thinking that it might make sense for these to be singletons, and it was useful in some code that was deleted, but this can be removed.